cpp/src/io/json/read_json.hpp

```diff
@@ -29,6 +29,19 @@
 
 namespace cudf::io::json::detail {
 
+// Some magic numbers
+constexpr int num_subchunks               = 10;  // per chunk_size
+constexpr size_t min_subchunk_size        = 10000;
+constexpr int estimated_compression_ratio = 4;
+constexpr int max_subchunks_prealloced    = 3;
```

Comment:
why not keep these in the source file?

Reply:
We need `max_subchunks_prealloced` in `read_json` (L307 in `read_json.cu`). Instead of including only this magic number in the header, I thought it would be better to have all of them in `read_json.hpp`.